public/app/core/components/NavBar/NavBarItemMenuTrigger.tsx

```diff
@@ -34,6 +35,12 @@ export function NavBarItemMenuTrigger(props: NavBarItemMenuTriggerProps): ReactE
   const ref = React.useRef<HTMLButtonElement>(null);
   const { menuTriggerProps, menuProps } = useMenuTrigger({}, state, ref);
 
+  useEffect(() => {
+    if (item.highlightId) {
+      reportExperimentView(`feature-highlights-${item.highlightId}-nav`, 'test', '');
```

Comment:
**Just an idea:** maybe in the future it would be nice to have an `enum` or a constant for the available experiment groups referenced from a single place

Reply:
Yes it's in Enterprise right now but I agree this should be moved into OSS. I'll do that in a separate PR.